bin/index.js

```diff
@@ -102,3 +127,18 @@ function defaultScannerCommand(name, options = {}) {
 
   return cmd;
 }
+
+function checkNodeSecureToken() {
+  if (!process.env.NODE_SECURE_TOKEN) {
+    console.log(
+      kleur
+        .white()
+        .bold(
+          `\n ${kleur
+            .yellow()
+            .bold(`Environment variable ${"\"NODE_SECURE_TOKEN\""} is missing.
+            \trun${"`npm login`"} to login`)}`
```

Comment:
Because there is no relation with npm login you need to remove the last part of the log

Reply:
Sure